Sources/AppcuesKit/Presentation/Extensions/View+Appcues.swift

```diff
@@ -107,7 +107,7 @@ extension View {
                     .padding(width)
                     .overlay(
                     // Need to adjust the corner radius to match the radius applied to the view.
-                    RoundedRectangle(cornerRadius: (style.cornerRadius ?? 0) - width / 2)
+                    RoundedRectangle(cornerRadius: max(0, (style.cornerRadius ?? 0) - width / 2))
```

Comment:
from my testing, it looked like we needed
```
cornerRadius + width / 2
```
actually, to match the radius of the view within that border?

given a border width 20, and a corner radius 10 on the view - this existing calculation gives:
![Screen Shot 2022-11-01 at 1 45 06 PM](https://user-images.githubusercontent.com/19266448/199302024-c7b3c29c-d3bd-494b-96cc-3bda848a126c.png)
with some visible gaps in the corners

updating to the new calculation gives
![Screen Shot 2022-11-01 at 1 45 42 PM](https://user-images.githubusercontent.com/19266448/199302136-37bfdbf4-3d57-4be7-942a-67801f992f4d.png)

using a `max` here would also result in no radius applied to the corners any time the width is larger than 2x the radius, for example a corner radius of 5 and a border width of 20.

Reply:
This scenario is now handled properly:
https://github.com/appcues/appcues-mobile-experience-spec/blob/1fbbd2d22985f74133f0e5002e0a54f48687ff17/component-examples/image-corner-10-border-30.json
https://github.com/appcues/appcues-mobile-experience-spec/blob/1fbbd2d22985f74133f0e5002e0a54f48687ff17/verification-suites/ios/SnapshotTests/__Snapshots__/ImageComponentTests/testCorner10Border30.1.png